test/expected/create_hypertable.out

```diff
@@ -37,6 +37,7 @@ select create_hypertable('test_schema.test_1dim', 'time');
  test_schema | test_table | table | postgres
 (2 rows)
 
+insert into test_schema.test_1dim VALUES ('2004-10-19 10:23:54+02',1.0);
```

Comment:
I don't think this file was updated to reflect the changes made in the test file? Looks like previous output